sdk/framework/backend.go

```diff
@@ -527,10 +527,11 @@ func (b *Backend) handleRootHelp(req *logical.Request) (*logical.Response, error
 	// the request, the type will be used as part of the request/response body
 	// names in the OAS document.
 	requestResponsePrefix := req.GetString("requestResponsePrefix")
+	genericPaths := req.Get("genericPaths").(bool)
```

Comment:
What happens if `genericPaths` isn't passed as a parameter? Does this panic on the type assertion?

Reply:
oops. yep. this one does. Let me fix that